### PR TITLE
client: python2 comapbility, pep8 fixes

### PR DESCRIPTION
**Pull request by MSusik**
Signed-off-by: mateuszsusik mateuszsusik@gmail.com

### CODE REVIEW

**Comment by MSusik** [comment]
I've tried two most popular binary packagers, and I couldn't obtain satisfactory results.

`pyinstaller` can't bind Gtk3 properly
`cx_Freeze` seem to have problems with Anaconda.

So it seems that scripts are the way to go.

File: client/main/dpcs [hunk 1]
```diff
@@ -1,5 +1,4 @@
-#! /usr/bin/env python
```
Comment (MSusik) [inline client/main/dpcs:1]: We don't support Python3

File: client/settings/dpcs-settingspanel/dpcs-settings [hunk 2]
```diff
@@ -1,146 +0,0 @@
-#!/usr/bin/env python3
```
Comment (MSusik) [inline client/settings/dpcs-settingspanel/dpcs-settings:1]: Moved directory up, and moved to Python2

File: client/setup.py [hunk 3]
```diff
@@ -32,10 +32,10 @@ def run_tests(self):
         'Topic :: System :: Software Distribution',
         'Topic :: Utilities'
       ],
-      scripts=['main/dpcs'],
```
Comment (MSusik) [inline client/setup.py:35]: No need to have two separate install files, settings should be installed always when dpcs is installed.

**Comment by bdfhjk** [comment]
Good work. Thanks for update! :+1:

**Pull request merged**